src/schema/partner_show.js

```diff
@@ -75,18 +74,21 @@ const PartnerShowType = new GraphQLObjectType({
           defaultValue: 25,
         },
       },
-      resolve: (show, options) => {
-        const path = `partner/${show.partner.id}/show/${show.id}/artworks`
-
-        let fetch = null
-
+      resolve: (
+        show,
+        options,
+        request,
+        { rootValue: { partnerShowArtworksLoader } }
+      ) => {
+        // TODO: Support fetching until an API is exhausted?
```

Comment:
Alright, this might be controversial. The `all` pattern is definitely not one to encourage. I looked in Force for all places where `all: true` might be getting passed in, and it's in the context of a show/fair booth artworks, as well as sale artworks.

These were being used in mobile and desktop Force artwork pages to show the module 'Other artworks in this...'.

So, if the artwork in question is at a show or fair booth, there's the section `Other artworks in this show/fair booth`, and shows all the works. If the artwork in question is at an auction, there's that section.

I set the size at 50 for `all`  here, and 99 (max our API will allow in 1 page) for the case of sales.

What do you think? I find it hard to imagine that that would be insufficient for loading that below-the-fold section...(besides those should probably be implemented via a inf scroll in that case).

Reply:
Why is this not paginating?

Reply:
Lol hit refresh

Reply:
Not sure I understand, are you saying I should refresh this PR to look for a comment that touches on this or that fetching all these records is related to refreshing? 😄

Reply:
Oh, I had a long-ish comment to basically explain dropping fetching all records in place of just one page (since fetching all in one resolver is an anti-pattern, those client views should be paginating/inf scroll), and that the places on Force where `all: true` was being used seemed droppable in support of this.

Reply:
I wonder if this is something for @sweir27 and @yuki24 to pick up, as this can’t be perceived as performant on the client side?

Reply:
In the meantime, I added a helper for `allViaLoader` which accepts any loader, and does the spread out fetching, so we can use that in the meantime.

But I would definitely open an issue for the follow up.

Reply:
@mzikherman I thought that we had removed the "all" behavior in the case of "other works in auctions" in favor of a button that redirects you to the sale page (since we could have 300 works in a sale!!). 

I support removing this functionality, but it's maybe worth identifying these places and talking to design about them. Happy to pick up this work with @yuki24 during this sprint, since it seems relevant.

Reply:
I re-added this functionality back (fetching all), and you're right that on desktop Force that seems to be the case. But, on mobile Force there still seems to be `all: true` in that context, and on both desktop and mobile Force in 'Other works from this show/fair booth' it appears to be used as well.